### PR TITLE
SRM - SLO downtime

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/service-reliability-management/slo-downtime/manage-slo-downtime.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+The SLO Downtime page displays a comprehensive list of all the SLO downtimes and also provides options to manage them.
+
+:::info note
+Currently, this feature is behind the feature flag `SRM_DOWNTIME`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+:::
+
+## SLO downtime page
+
+The SLO downtime page displays information about all the configured SLO downtimes.
```
Comment (aimurphy) [inline docs/service-reliability-management/slo-downtime/manage-slo-downtime.md:15]:
```suggestion
The SLO Downtime page displays information about all the configured SLO downtimes.
```

File: docs/service-reliability-management/slo-downtime/manage-slo-downtime.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+To edit a SLO downtime:
+
+1. Select the pencil icon next to the SLO downtime that you want to edit.  
+   The Edit SLO Downtime page appears.
+2. Make desired changes to the SLO downtime configuration and select **Save** to save the changes.
+
+
+## Delete a SLO downtime
+
+To delete a SLO downtime, select the delete icon next to it, and in the confirmation message that appears, select **Delete**.
```
Comment (aimurphy) [inline docs/service-reliability-management/slo-downtime/manage-slo-downtime.md:82]:
```suggestion
To delete a SLO downtime, select the delete icon next to it, and, in the confirmation message that appears, select **Delete**.
```

**Review by aimurphy — APPROVED** [review]
approved w some suggestions

**Comment by SudheendraKatte** [comment]
Committed the review suggestions and merging.

**Pull request merged**